core/src/com/unciv/ui/newgamescreen/GameOptionsTable.kt

```diff
@@ -95,9 +96,13 @@ class GameOptionsTable(
             { gameParameters.oneCityChallenge = it }
 
     private fun Table.addNuclearWeaponsCheckbox() =
-            addCheckbox("Enable nuclear weapons", gameParameters.nuclearWeaponsEnabled)
+            addCheckbox("Enable Nuclear Weapons", gameParameters.nuclearWeaponsEnabled)
```

Comment:
We need to be careful with changes like these to avoid losing all existing translations - I'll deal with it on the next version release but this is something we need to be aware of

Reply:
I remember my idea to save such xlations... But that only shows fixing case for strings should, when possible, be separate PR's also fixing the translations, and be submitted right after a release..